packages/react/package.json

```diff
@@ -64,7 +64,9 @@
     "color2k": "^2.0.2",
     "css-loader": "^6.7.4",
     "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.7.0",
```

Comment:
Is adding this here intentional and/or needed? This is already set in the monorepo root, so this is a duplication. If you do need it, could you hoist it?

Reply:
Not required sorry, must have slipped through, good spot 👍